exasol/python_extension_common/deployment/language_container_deployer.py

```diff
@@ -203,8 +208,17 @@ def run(
                             For this to work either of the two conditions should be met.
                             The pyexasol connection should have an open schema, or
                             The calling user should have a permission to create schema.
+        print_alter_session_activation - If True and alter_system is False,
+                                         it will print the ALTER SESSION command to stdout.
         """
 
+        if alter_system and print_alter_session_activation:
+            logging.warning(
```

Comment:
I think, we don't need the warning and simply should explain in the docstring that the printing only happens when alter system false